packages/web/webpack.config.js

```diff
@@ -75,18 +79,13 @@ module.exports = (_, argv) => {
         template: path.join(__dirname, 'index.html'),
       }),
       new webpack.DefinePlugin({
-        ...(isDevelopment ? { 'process.env': JSON.stringify(dotenv.config().parsed) } : {}),
+        ...(isDevelopment
+          ? {
+              'process.env': env,
+            }
+          : {}),
       }),
-      ...(isDevelopment
-        ? []
-        : [
-            new webpack.EnvironmentPlugin({
-              API_ORIGIN: 'https://staging.api.gradejs.com',
-              GA_ID: '',
-              PLAUSIBLE_DOMAIN: '',
-              DUMP_ANALYTICS: '',
-            }),
-          ]),
+      ...(isDevelopment ? [] : [new webpack.EnvironmentPlugin(env)]),
```

Comment:
It seems like we can use `EnvironmentPlugin(env)` for both development and production